server/src/main/java/io/crate/execution/engine/aggregation/impl/NumericSumAggregation.java

```diff
@@ -182,7 +182,10 @@ public DocValueAggregator<?> getDocValueAggregator(LuceneReferenceResolver refer
                                                        List<Reference> aggregationReferences,
                                                        DocTableInfo table,
                                                        List<Literal<?>> optionalParams) {
-        Reference reference = aggregationReferences.get(0);
+        Reference reference = aggregationReferences.getFirst();
```

Comment:
Out of curiosity - are these changes triggered by some intellij warning or suggestion?

Strictly speaking these are not merely style changes but also change semantics. (Trading a `IndexOutOfBoundsException` with a `NoSuchElementException`.

I don't think it matters in these places, but I'd tend to avoid such changes as they're unnecessary

Reply:
No, this was not triggered by any warnings. You have a valid point with the change semantics. I will revert this.